docs/v3.x/plugins/users-permissions.md

```diff
@@ -484,88 +484,100 @@ Response payload:
+8. If the request contains the code contained in the link at step 3., the password is updated
+9. The user can log in with the new password
+
+In the following section we will detail steps 3. and 7..
+
+#### Forgotten password: ask for the reset password link
+
+This action sends an email to a user with the link to your own reset password page.
+The link will be enriched with the url param `code` that is needed for the [reset password](#reset-password) at step 7..
+
+First, you must specify the url to your reset password page in the admin panel: **Roles & Permissions > Advanced Settings > Reset Password Page**.
 
-- `email` is your user email.
+Then, your **forgotten password page** has to make the following request to your backend.
 
 ```js
 import axios from 'axios';
 
 // Request API.
 axios
   .post('http://localhost:1337/auth/forgot-password', {
-    email: 'user@strapi.io',
+    email: 'user@strapi.io', // user's email
   })
   .then(response => {
-    // Handle success.
     console.log('Your user received an email');
   })
   .catch(error => {
-    // Handle error.
     console.log('An error occurred:', error.response);
   });
 ```
 
-This action will send the user an email that contains a URL with the needed code for the [reset password](#reset-password).
-The URL must link to your reset password form in your frontend application.
+#### Reset Password: send the new password
 
-To configure it you will have to go in the Roles & Permissions settings and navigate to the Advanced Settings tab.
+This action will update the user password.
+Also works with the [GraphQL Plugin](./graphql.md), with the `resetPassword` mutation.
 
-### Reset Password
-
-This action will reset the user password.
-Also works with the [GraphQL Plugin](./graphql.md), exposes `resetPassword` mutation.
-
-#### Usage
-
-- `code` is the url params received from the email link (see forgot password)
+Your **reset password page** has to make the following request to your backend.
 
 ```js
 import axios from 'axios';
 
 // Request API.
 axios
   .post('http://localhost:1337/auth/reset-password', {
-    code: 'privateCode',
-    password: 'myNewPassword',
-    passwordConfirmation: 'myNewPassword',
+    code: 'privateCode', // code contained in the reset link of step 3.
+    password: 'userNewPassword',
+    passwordConfirmation: 'userNewPassword',
   })
   .then(response => {
-    // Handle success.
     console.log("Your user's password has been reset.");
   })
   .catch(error => {
-    // Handle error.
     console.log('An error occurred:', error.response);
   });
 ```
 
+Congrats, you're done!
+
 ### Email validation
 
-This action sends an email to the user with the link to confirm the user.
+:::tip NOTE
+In production, make sure the `url` config property is set. Otherwise the validation link will redirect to `localhost`. More info on the config [here](../concepts/configurations.html#server).
```

Comment:
links should be pointing to .md files. vue-press generate the right urls on build